meta-mender-qemu/recipes-support/gnutls/gnutls_%.bbappend

```
@@ -1,3 +1,3 @@
 
-PACKAGECONFIG_append = " p11-kit"
+PACKAGECONFIG_append_mender-client-install = " p11-kit"
```

The problem with the integration test is that `p11tool` is not found in the image; in other words, `p11-kit` is not installed.

I fixed it locally with the following modification:
```suggestion
ACKAGECONFIG_append = "${@bb.utils.contains('DISTRO_FEATURES', 'mender-client-install', ' p11-kit', '', d)}"
```

I don't have a good reason as per why this change fixes the issue, I followed a bit the logic that I have seen in other places where we append something to `PACKAGECONFIG`.

@kacf @drewmoseley Does this make sense to you?

---

Yes, it does!

---

I'll submit a new PR with this fix and conflicts resolution from master.